src/entities/People.js

```diff
@@ -1,26 +1,30 @@
-import { EntitySchema } from 'typeorm';
+import {EntitySchema} from 'typeorm';
 
 const People = new EntitySchema({
   name: 'People',
   columns: {
     id: {
       primary: true,
-      type: 'int',
-      generated: true,
-    },
-    name: {
       type: 'varchar',
+      generated: true,
     },
-    height: {
-      type: 'int',
-    },
-    mass: {
-      type: 'int',
-    },
+    name: {type: 'varchar'},
+    height: {type: 'int'},
+    mass: {type: 'int'},
+    hair_color: {type: 'varchar'},
+    skin_color: {type: 'varchar'},
+    eye_color: {type: 'varchar'},
+    birth_year: {type: 'varchar'},
+    gender: {type: 'varchar'},
+    homeworld: {type: 'varchar'},
+    films: {type: 'simple-array'},
+    species: {type: 'simple-array'},
+    vehicles: {type: 'simple-array'},
+    starships: {type: 'simple-array'},
```

Comment:
Do not forget to change films', species', vehicles' and starships' type to Many-To-Many relation.
https://orkhan.gitbook.io/typeorm/docs/many-to-many-relations